src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -44,8 +44,13 @@
 
 Welcome to the cool new Azure CLI!
 
-Use `az --version` to display the current version.
-Here are the base commands:
+Documentation: https://docs.microsoft.com/en-us/cli/azure
```

Comment:
pls use short url aka.ms/cli

Reply:
Short link is difficult to maintain, given the conditional that https://aka.ms can't use an email group as short link owner. Let's discuss in more detail about short links.